repair/featurize/freqfeat.py

```diff
@@ -29,3 +30,6 @@ def create_tensor(self):
         tensors = [self.gen_feat_tensor(res, self.classes) for res in results]
         combined = torch.cat(tensors)
         return combined
+
+    def feature_names(self):
+        return self.all_attrs
```

Comment:
It might be a good idea to prepend `'freq_'` before attribute names, e.g., return `[ 'freq_' + attr for attr in self.all_attrs ]`